src/general/disallowanonymous/src.rego

```diff
@@ -2,18 +2,51 @@ package k8sdisallowanonymous
 
 violation[{"msg": msg}] {
   not is_allowed(input.review.object.roleRef, input.parameters.allowedRoles)
```

Comment:
We should get this PR merged first: https://github.com/open-policy-agent/gatekeeper-library/pull/578

Reply:
Merged!